react/README.md

```diff
@@ -531,7 +531,7 @@
 
     // good
     render() {
-      return (<div />);
```

Comment:
single-line jsx are absolutely allowed to have parens - and i personally encourage it always - the guide only requires it around multiline parens. Please revert.